src/libs/actions/Wallet.js

```diff
@@ -377,6 +377,47 @@ function activateWallet(currentStep, parameters) {
         });
 }
 
+/**
+ * Complete the "Accept Terms" step of the wallet activation flow.
+ *
+ * @param {Object} parameters
+ * @param {Boolean} parameters.hasAcceptedTerms
+ */
+function acceptWalletTerms(parameters) {
+    const optimisticData = [
```

Comment:
Looks like we use walletTerms.loading here: https://github.com/Expensify/App/blob/e9d29b12e27b57138fb225b67054e508d3274766/src/pages/EnablePayments/TermsStep.js#L119

I guess we don't really need it since the activation success view will be optimistically shown?

Reply:
Ah yeah, good point. Looks like it's the only place `loading` was used, so I'll just remove the prop entirely.